tests/sentry/test_sentry_sdk_2x.py

```diff
@@ -0,0 +1,129 @@
+"""Test fixtures copied from https://github.com/getsentry/sentry-python/
+"""
+
+import uuid
+import importlib.metadata
+import pytest
+from logging import getLogger
+from sentry_sdk.tracing import Transaction
+
+from .utils import get_transaction_trace_id, get_captured_transactions, assert_jrpc_batch_sentry_items
```

Comment:
Lets move utils.py to fastapi_jsonrpc/contrib/sentry/test_utils.py

and do not use `tests` folder like a python package

Reply:
for some reason otherwise I get error here
https://github.com/smagafurov/fastapi-jsonrpc/blob/c3176e6415205843d013a3bed221c39a1bcbb09d/fastapi_jsonrpc/__init__.py#L1248

```
AttributeError: 'NoneType' object has no attribute '__name__'
```